server/e2e/integration_item_test.go

```diff
@@ -128,3 +149,166 @@ func TestIntegrationItemListAPI(t *testing.T) {
+	r.Value("fields").Equal([]interface{}{
+		map[string]string{
+			"id":    fId.String(),
+			"type":  "text",
+			"value": "test value",
+		},
+	})
+	r.Value("modelId").Equal(mId.String())
+	r.Value("refs").Equal([]string{"latest"})
+}
+
+// Post|/items/{itemId}
+func TestIntegrationUpdateItemAPI(t *testing.T) {
+	e := StartServer(t, &app.Config{}, true, baseSeeder)
+
+	e.POST("/api/models/{modelId}/items", id.NewModelID()).
+		Expect().
+		Status(http.StatusUnauthorized)
+
+	e.POST("/api/models/{modelId}/items", id.NewModelID()).
+		WithHeader("authorization", "secret_abc").
+		Expect().
+		Status(http.StatusUnauthorized)
+
+	e.POST("/api/models/{modelId}/items", id.NewModelID()).
+		WithHeader("authorization", "Bearer secret_abc").
+		Expect().
+		Status(http.StatusUnauthorized)
+
+	e.POST("/api/models/{modelId}/items", id.NewModelID()).
+		WithHeader("authorization", "Bearer secret_1234567890").
+		Expect().
+		Status(http.StatusBadRequest)
+
+	r := e.POST("/api/models/{modelId}/items", mId).
+		WithHeader("authorization", "Bearer secret_1234567890").
+		WithJSON(map[string]interface{}{
+			"fields": []interface{}{
+				map[string]string{
+					"id":    fId.String(),
+					"value": "test value",
+				},
+			},
+		}).
+		Expect().
+		Status(http.StatusOK).
+		JSON().
+		Object()
+	r.Keys().
+		Contains("id", "modelId", "fields", "createdAt", "updatedAt", "version", "parents", "refs")
+	r.Value("fields").Equal([]interface{}{
+		map[string]string{
+			"id":    fId.String(),
+			"type":  "text",
+			"value": "test value",
+		},
+	})
+	r.Value("modelId").Equal(mId.String())
+	r.Value("refs").Equal([]string{"latest"})
+}
+
+// Get|/items/{itemId}
+func TestIntegrationGetItemAPI(t *testing.T) {
+	e := StartServer(t, &app.Config{}, true, baseSeeder)
+
+	e.GET("/api/items/{itemId}", id.NewItemID()).
+		Expect().
+		Status(http.StatusUnauthorized)
+
+	e.GET("/api/items/{itemId}", id.NewItemID()).
+		WithHeader("authorization", "secret_abc").
+		Expect().
+		Status(http.StatusUnauthorized)
+
+	e.GET("/api/items/{itemId}", id.NewItemID()).
+		WithHeader("authorization", "Bearer secret_abc").
+		Expect().
+		Status(http.StatusUnauthorized)
+
+	e.GET("/api/items/{itemId}", id.NewItemID()).
+		WithHeader("authorization", "Bearer secret_1234567890").
+		Expect().
+		Status(http.StatusNotFound)
+
+	e.GET("/api/items/{itemId}", itmId).
+		WithHeader("authorization", "Bearer secret_1234567890").
+		Expect().
+		Status(http.StatusOK).
+		JSON().
+		Object().Keys().
+		Contains("id", "modelId", "fields", "createdAt", "updatedAt", "version", "parents", "refs")
+}
+
+// Delete|/items/{itemId}
+func TestIntegrationDeleteItemAPI(t *testing.T) {
+	e := StartServer(t, &app.Config{}, true, baseSeeder)
+
+	e.DELETE("/api/items/{itemId}", itmId).
+		WithHeader("authorization", "Bearer secret_abc").
+		Expect().
+		Status(http.StatusUnauthorized)
+
+	e.GET("/api/items/{itemId}", itmId).
+		WithHeader("authorization", "Bearer secret_1234567890").
```

Comment:
```suggestion
		WithHeader("authorization", "Bearer" + secret).
```

Can you please make the correct token a constant?